viewer/packages/360-images/src/Image360Icon.ts

```diff
@@ -53,25 +106,25 @@ export class Image360Icon extends THREE.Group {
 
     function drawOuterCircle() {
       context.beginPath();
-      context.lineWidth = 8;
+      context.lineWidth = 16;
       context.strokeStyle = '#FFFFFF';
-      context.arc(textureSize / 2, textureSize / 2, textureSize / 2 - 4, 0, 2 * Math.PI);
+      context.arc(textureSize / 2, textureSize / 2, textureSize / 2 - 8, 0, 2 * Math.PI);
       context.stroke();
     }
 
```

Comment:
Personally, I'd like to see these magical constant getting computed explicitly from the texture size. Goes for the constants further down in this file as well